__tests__/routes/ResetPassword.test.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { render, fireEvent, waitFor } from '@testing-library/react';
+import RecoverPasswordForm from '@/components/RecoverPassword/RecoverPasswordForm';
+import { resetPasswordForEmail } from '@/lib/actions/auth.client';
```

Comment:
I ran the tests with Jacob @jvniorrr , It passed and the logic is correct though the UI will give a pass in some situations due to the UI logic and not the test